### PR TITLE
Fixtures load

### PR DESCRIPTION
**Pull request by EVDW**

### CODE REVIEW

File: _includes/_components/fixtures-load.html [hunk 1]
```diff
@@ -1,50 +1,80 @@
...
             <h2>Fixtures Load</h2>
-            <p>This component blablabla</p>
-          </div>
-        </header>
-        <div class="container content">
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-          </p>
-          <div class="row">
+            <p>One command allows quick to load your fake data created with nelmio/alice library.</p>
```
Comment (einenlum) [inline _includes/_components/fixtures-load.html]: `Allows you to use nelmio/alice library when loading your fixtures.`

File: _includes/_components/fixtures-load.html [hunk 2]
```diff
@@ -1,50 +1,80 @@
...
     {
-        return $subject->doSomething();
+        $user = new User();
+        $user->setUsername('MD');
+        $user->setFullname('Martin Dupond');
+        $user->setBirthday('01/01/1980');
+        $user->setEmail('martin.dupond@mail.com');
+
+        $manager->persist($user);
+        $manager->flush();
```
Comment (einenlum) [inline _includes/_components/fixtures-load.html:39]: I would set an example with several users, to show how it's easy with Alice.

File: _includes/_components/fixtures-load.html [hunk 3]
```diff
@@ -1,50 +1,80 @@
...
+class LoadUserData implements FixtureInterface
 {
-    public function test(SomeClass $subject)
+    public function load(ObjectManager $manager)
     {
-        return $subject->doSomething();
+        $user = new User();
+        $user->setUsername('MD');
+        $user->setFullname('Martin Dupond');
+        $user->setBirthday('01/01/1980');
```
Comment (einenlum) [inline _includes/_components/fixtures-load.html]: It's a datetime in your alice example, not a string.

File: _includes/_components/fixtures-load.html [hunk 4]
```diff
@@ -1,50 +1,80 @@
...
-                <div class="banner"></div>
-{% highlight php %}
-<?php
+                <div class="rad">
+                    <div class="banner"></div>
+{% highlight yml %}
+# Resources/fixtures/orm/users.yml
+
+App\Entity\User:
+    user:
```
Comment (einenlum) [inline _includes/_components/fixtures-load.html]: See my comment above. I would show how easy you can generate multiple users.
`user_{1..10}:`

File: _includes/_components/fixtures-load.html [hunk 5]
```diff
@@ -1,50 +1,80 @@
...
-    }
-}
 {% endhighlight %}
-              </div>
+                </div>
+
+                <div class="standard">
+{% highlight sh %}
+
+$ php app/console rad:fixtures:load
```
Comment (einenlum) [inline _includes/_components/fixtures-load.html:95]: maybe also add `php bin/console rad:fixtures:load` if symfony3 is used.

File: _includes/_components/fixtures-load.html [hunk 6]
```diff
@@ -24,24 +24,53 @@
...
     public function load(ObjectManager $manager)
     {
-        $user = new User();
-        $user->setUsername('MD');
-        $user->setFullname('Martin Dupond');
-        $user->setBirthday('01/01/1980');
-        $user->setEmail('martin.dupond@mail.com');
+        for ($i = 0; $i < 10; $i++) {
+            $timestamp = mt_rand(1, time());
+            $randomDate = new DateTime('@'.$timestamp);
```
Comment (Einenlum) [inline _includes/_components/fixtures-load.html]: I think you should create a private method to generate a random date.

File: _includes/_components/fixtures-load.html [hunk 7]
```diff
@@ -24,24 +24,53 @@
...
+
+    private function generateUsername()
+    {
+        $usernames = [
+            'nrats',
+            'nrstauie',
+            'nrpuldv'
+        ];
+
+        return array_rand($usernames, 1);
```
Comment (einenlum) [inline _includes/_components/fixtures-load.html]: second parameter is useless here (1 is the default value)

File: _includes/_components/fixtures-load.html [hunk 8]
```diff
@@ -24,24 +24,53 @@
...
+
+    private function generateEmail()
+    {
+        $mails = [
+            'nrats@mail.com',
+            'nrstauie@mail.com',
+            'nrpuldv@mail.com'
+        ];
+
+        return array_rand($mails, 1);
```
Comment (einenlum) [inline _includes/_components/fixtures-load.html]: second parameter is useless here (1 is the default value)

**Pull request merged**